frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -2707,7 +2707,7 @@ export const widgets = [
       },
       defaultValue: {
         type: 'code',
-        displayName: 'Default Value',
+        displayName: 'Default value',
```

Comment:
Hey @devmusiczz, please revert this change; this is not the property `Bounded Box`